converters/fromZigbee.js

```diff
@@ -1000,35 +1012,48 @@ const converters = {
         type: ['commandStep', 'commandStepWithOnOff'],
         convert: (model, msg, publish, options, meta) => {
             const direction = msg.data.stepmode === 1 ? 'down' : 'up';
-            const payload = {
+            let payload = {
                 action: postfixWithEndpointName(`brightness_step_${direction}`, msg, model),
                 action_step_size: msg.data.stepsize,
                 action_transition_time: msg.data.transtime / 100,
             };
-            addActionGroup(payload, msg, model);
 
             if (options.simulated_brightness) {
-                let brightness = globalStore.getValue(msg.endpoint, 'simulated_brightness_brightness', 255);
+                const keys = getSimulatedBrightnessKeys(options, msg, model);
+                let brightness = globalStore.getValue(msg.endpoint, keys.brightness, 255);
                 const delta = direction === 'up' ? msg.data.stepsize : -1 * msg.data.stepsize;
                 brightness += delta;
                 brightness = numberWithinRange(brightness, 0, 255);
-                globalStore.putValue(msg.endpoint, 'simulated_brightness_brightness', brightness);
+                globalStore.putValue(msg.endpoint, keys.brightness, brightness);
                 payload.brightness = brightness;
+
+                if (options.simulated_brightness.multi_endpoint) {
+                    const property = postfixWithEndpointName('brightness', msg, model);
+                    payload = {[property]: brightness};
+                }
             }
 
+            addActionGroup(payload, msg, model);
             return payload;
         },
     },
     command_stop: {
         cluster: 'genLevelCtrl',
         type: ['commandStop', 'commandStopWithOnOff'],
         convert: (model, msg, publish, options, meta) => {
+            let payload = {action: postfixWithEndpointName(`brightness_stop`, msg, model)};
             if (options.simulated_brightness) {
-                clearInterval(globalStore.getValue(msg.endpoint, 'simulated_brightness_timer'));
-                globalStore.putValue(msg.endpoint, 'simulated_brightness_timer', undefined);
+                const keys = getSimulatedBrightnessKeys(options, msg, model);
+                clearInterval(globalStore.getValue(msg.endpoint, keys.timer));
+                globalStore.putValue(msg.endpoint, keys.timer, undefined);
+
+                if (options.simulated_brightness.multi_endpoint) {
+                    const property = postfixWithEndpointName('brightness', msg, model);
+                    const brightness = globalStore.getValue(msg.endpoint, keys.brightness, 255);
+                    payload = {[property]: brightness};
```

Comment:
If I understand correctly, with this the `action` will be removed from the `payload`?